### PR TITLE
IIIF thumbnails. Fixes #216

### LINKED ISSUE #216: Use iiif for the thumbnails.
Take a look at these commits:
https://github.com/curationexperts/alexandria/commit/91f3c61346f99b10b8b374599e1b34c56de2f81b
https://github.com/curationexperts/alexandria/commit/c5c6693a55a160cbcf5ddcbd898a3e67d2d3ebfe
https://github.com/curationexperts/alexandria/commit/e5c9fb6bc9239c1b4069e599cd12c17a2ac5a420

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Rebase, pls.

File: app/services/iiif_thumbnail_path_service.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class IIIFThumbnailPathService < Sufia::WorkThumbnailPathService
+  class << self
+    # TODO: we need to override this method until https://github.com/projecthydra/curation_concerns/pull/843 is merged
```
Comment (mjgiarlo) [inline app/services/iiif_thumbnail_path_service.rb]: That PR is merged.

File: app/views/curation_concerns/file_sets/media_display/_image.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<% # TODO: we can remove this partial when https://github.com/projecthydra/curation_concerns/pull/844 is merged %>
```
Comment (mjgiarlo) [inline app/views/curation_concerns/file_sets/media_display/_image.html.erb]: projecthydra/curation_concerns#844 is merged.

Comment (jcoyne) [inline app/views/curation_concerns/file_sets/media_display/_image.html.erb]: Yes, but not released yet.

**Pull request merged**